pgjdbc/src/testFixtures/java/org/postgresql/test/TestUtil.java

```diff
@@ -740,10 +743,11 @@ public static String escapeString(Connection con, String value) throws SQLExcept
     if (con == null) {
       throw new NullPointerException("Connection is null");
     }
-    if (con instanceof PgConnection) {
-      return ((PgConnection) con).escapeString(value);
-    }
-    return value;
+    return Try
+        .call(() -> con.unwrap(PgConnection.class))
+        .andThenTry(it -> it.escapeString(value))
+        .toOptional()
+        .orElse(value);
```

Comment:
Frankly speaking, this is a questionable refactoring. In the long run we would rather use Java's `if (con instanceof PgConnection pgCon) { pgCon.escapeString(...`, so using `Try` does not feel suitable here.

It is funny we can't use regular `Optional.map` here as `escapeString` throws SQLException.

Reply:
You know, yes, probably, I agree. I'll roll this back.

Reply:
rolled back